### PR TITLE
sqlite2csv

### PR DESCRIPTION
**Pull request by ashishpriyadarshiCIC**

### CODE REVIEW

**Comment by henrykironde** [comment]
Why do we have many commits?

File: retriever/lib/engine_tools.py [hunk 1]
```diff
@@ -131,6 +134,17 @@ def json2csv(input_file, output_file=None, header_values=None, encoding=ENCODING
     subprocess.call(['rm', '-r', input_file])
     return output_file
 
+def sqlite2csv(database):
```
Comment (apoorvaeternity) [inline retriever/lib/engine_tools.py:137]: Follow PEP8: https://www.python.org/dev/peps/pep-0008/

Comment (ashishpriyadarshiCIC) [reply retriever/lib/engine_tools.py:137]: Will do @apoorvaeternity :+1:

Comment (ashishpriyadarshiCIC) [reply retriever/lib/engine_tools.py:137]: @apoorvaeternity Should I check my changes if following PEP8 using pycodestyle?

Comment (ashishpriyadarshiCIC) [reply retriever/lib/engine_tools.py:137]: @apoorvaeternity  Sorry, Didn't know pycodestyle was the replacement for PEP8 in future Python releases. Made the changes in the code according to PEP8.

**Comment by henrykironde** [comment]
Ref: #1505

**Pull request closed without merging**